docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+
+  mariadb:
+    build: docker/database
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: 'mdp-lib'
+    ports:
+      - "52000:3306"
```

Comment:
This exposes mariadb to the local machine; I don't think that's necessary by default

Reply:
Now I find it useful to be able to talk to the database directly, especially while futzing with the schema files. I'll comment that out and leave a note.